### ISSUE TITLE
Provide attributes of NUMA nodes

### ISSUE THREAD

**Issue by j-fu** [JuliaParallel/Hwloc.jl#29]
Hi, 
it would be interesting to obtain the attributes of the NUMA nodes from Julia. In particular, memory size may be  important. 
I looked at the implementation, it doesn't seem too hard to do. Probably someone who wrote the wrapper could be quick on this. Therefore this issue instead of another PR.  But let me know...

**Comment by carstenbauer** [comment]
For memory size, see the first step in #46 and the corresponding issue #45.

Regarding NUMA nodes, I have to figure out why Hwloc.jl tells me that there are 2 `:Package`s but 0 `:NUMANode`s on my supercomputer nodes. Note that `lstopo` and `hwloc-info` lists the two NUMA nodes. So maybe there is something wrong in the wrapper.

**Comment by carstenbauer** [comment]
Probably off-topic for this issue but it might still interest you. I'm currently trying to improve the topology printing and so on to also display cache sizes and (once #46 is merged) memory information (similar to what `lstopo` does). There also is a small summary at the end.

From [HwlocViz.jl](https://github.com/crstnbr/HwlocViz.jl):
```Julia
julia> print_topology()
Machine
└─ Package L#0 P#0
   └─ L3 (12 MB)
      ├─ L2 (256 KB)
      │  └─ L1 (32 KB)
      │     └─ Core L#0 P#0
      │        ├─ PU L#0 P#0
      │        └─ PU L#1 P#1
      ├─ L2 (256 KB)
      │  └─ L1 (32 KB)
      │     └─ Core L#1 P#1
      │        ├─ PU L#2 P#2
      │        └─ PU L#3 P#3
      ├─ L2 (256 KB)
      │  └─ L1 (32 KB)
      │     └─ Core L#2 P#2
      │        ├─ PU L#4 P#4
      │        └─ PU L#5 P#5
      ├─ L2 (256 KB)
      │  └─ L1 (32 KB)
      │     └─ Core L#3 P#3
      │        ├─ PU L#6 P#6
      │        └─ PU L#7 P#7
      ├─ L2 (256 KB)
      │  └─ L1 (32 KB)
      │     └─ Core L#4 P#4
      │        ├─ PU L#8 P#8
      │        └─ PU L#9 P#9
      └─ L2 (256 KB)
         └─ L1 (32 KB)
            └─ Core L#5 P#5
               ├─ PU L#10 P#10
               └─ PU L#11 P#11

Packages: 1 	 L3: 1 (12 MB)
Cores: 6 	 L2: 6 (256 KB)
PUs: 12 	 L1: 6 (32 KB)
```
